.eslintrc.json

```diff
@@ -0,0 +1,29 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "env": {
+    "browser": true,
+    "commonjs": true,
+    "es2021": true,
+    "jest": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:@typescript-eslint/strict"
+  ],
+  "parserOptions": {
+    "project": "./tsconfig.json",
+    "ecmaVersion": 2020,
+    "sourceType": "module"
```

Comment:
I didn't find this option in the parser's documentation: https://typescript-eslint.io/packages/parser/

What is it? Where did you find it?

Reply:
My bad, it's already set as default